apps/api/src/domain/games/memory-match/queries/memory-match-queries.fetch-many.e2e.spec.ts

```diff
@@ -122,4 +157,86 @@ describe(`when querying for a memory match round: fetch many`, () => {
             });
         });
     });
+
+    describe(`when the user is authenticated as a viewer`, () => {
+        beforeAll(async () => {
+            await setItUp(new CoscradUserWithGroups(nonAdminUser, []));
+        });
+
+        describe(`when no filters are provided`, () => {
+            it(`should return all published rounds`, async () => {
+                const res = await request(app.getHttpServer()).get(indexEndpoint);
+
+                expect(res.status).toBe(HttpStatusCode.ok);
+
+                const {
+                    body: { entities },
+                } = res;
+
+                // there should be two rounds visible to authenticated user
+                expect(entities).toHaveLength(1);
+
+                expect(entities[0].id).toBe(publishedRound.id);
+
+                // this is a contract test to ensure we don't break the client
+                expect(entities).toMatchSnapshot();
+            });
+        });
+    });
+
+    describe(`when the user is authenticated as a coscrad admin`, () => {
+        beforeAll(async () => {
+            await setItUp(new CoscradUserWithGroups(coscradAdmin, []));
+        });
+
+        describe(`when no filters are provided`, () => {
+            it(`should return all published and unpublished rounds`, async () => {
+                const res = await request(app.getHttpServer()).get(indexEndpoint);
+
+                expect(res.status).toBe(HttpStatusCode.ok);
+
+                const {
+                    body: { entities },
+                } = res;
+
+                // there should be two rounds visible to authenticated user
```

Comment:
Shouldn't we reword this to an admin? Authenticated is not the same as authorized. The viewer is authenticated, but cannot access private rounds.